implementation/contracts/deposit/Deposit.sol

```diff
@@ -28,7 +28,9 @@ contract Deposit is DepositFactoryAuthority {
     /* solium-disable-next-line no-empty-blocks */
     constructor () public {}
 
-    function () external payable {}
+    function () external payable {
+        require(msg.sender == self.keepAddress, "Deposit contract can only receive ETH from bondedECDSAKeep");
```

Comment:
How about just “from underlying keep”? Doesn't seem like we use `bondedECDSAKeep` specifically anywhere, so it's a weird single-use term.

Won't block on this obv.